crates/bevy_tasks/src/single_threaded_task_pool.rs

```diff
@@ -59,7 +59,7 @@ impl TaskPool {
     /// This is similar to `rayon::scope` and `crossbeam::scope`
     pub fn scope<'scope, F, T>(&self, f: F) -> Vec<T>
     where
-        F: FnOnce(&mut Scope<'scope, T>) + 'scope + Send,
+        F: FnOnce(&mut Scope<'scope, T>) + 'scope,
```

Comment:
This makes the single threaded and multi threaded task pool not have the same function signature.

Reply:
No? I removed this unnecessary Send bound on both versions.

Reply:
Right, my bad. Looked at a single commit.